.ci/build_cli.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -eE
```

Comment:
You may need also `set -o pipefail` for robustness

Reply:
Please add clarification for each remarks

Reply:
Pipe was removed